### PR TITLE
Add issue and pull request templates

### LINKED ISSUE #328: Create issue (and PR) templates
In order to get better issue descriptions, we should use issue (and PR) templates: https://github.com/blog/2111-issue-and-pull-request-templates

### PR DESCRIPTION
**Pull request by superbock**
## Changes proposed in this pull request

Add templates for issues and pull requests.

This pull request fixes #328.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## Changes proposed in this pull request
+
+Please provide a detailled description of the changes proposed in this pull
```
Comment (fdlm) [inline .github/pull_request_template.md]: "detailed"....

**Comment by fdlm** [comment]
Maybe it would be nice to have a reminder of what to do before a pull request (run pep8 and unit tests)?

**Comment by superbock** [comment]
Good point! But maybe this should go into a contributing document (https://help.github.com/articles/setting-guidelines-for-repository-contributors/)?

**Comment by fdlm** [comment]
Probably, yes.

**Pull request merged**